bin/rpm-s3

```diff
@@ -272,6 +284,7 @@ if __name__ == '__main__':
     parser.add_option('-d', '--delete', action='store_true', default=False)
     parser.add_option('-r', '--region')
     parser.add_option('-c', '--host', default='s3-eu-central-1.amazonaws.com')
+    parser.add_option('-a', '--archs', type='string', default='', action='callback', callback=get_comma_separated_args)
```

Comment:
nit: `default=''` works but I don't think it's the best solution. With this solution the value of `options.archs` when the option is not passed is `''` (and not `[]` as I would have expected). 

The relevant loop then iterates over the characters of the empty string (so it's equivalent to having an empty list in that sense) but it could be confusing.

Reply:
Indeed, I fixed it